apps/storefront/src/utils/b3CurrencyFormat.ts

```diff
@@ -13,10 +13,7 @@ interface MoneyFormat {
 }
 
 export const currencyFormatInfo = () => {
-  const currentCurrency =
-    import.meta.env.VITE_IS_LOCAL_ENVIRONMENT === 'TRUE'
```

Comment:
We don't need this validation to make a difference between local and other envs, we can use the value from the activeCurrencyInfo